spring-petclinic-api-gateway/src/main/resources/static/scripts/pet-form/pet-form.template.html

```diff
@@ -4,7 +4,7 @@ <h2>Pet</h2>
     <div class="form-group">
         <label class="col-sm-2 control-label">Owner</label>
         <div class="col-sm-6">
-            <p class="form-control-static">{{$ctrl.pet.owner}}</p>
+            <p class="form-control-static">{{$ctrl.pet.owner.name}}</p>
```

Comment:
I guess owner will have other properties than `name`. What's the reason for the change here?

Reply:
reverted